src/sos.pc.in

```diff
@@ -0,0 +1,18 @@
+## Sandia OpenSHMEM package metadata
+## Requires pkg-config 0.28 or later
+
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: sos
+Description: @PACKAGE_NAME@
+URL: @PACKAGE_URL@
+Version: @VERSION@
+Libs: -L${libdir} @WRAPPER_COMPILER_EXTRA_LDFLAGS@ -lsma @WRAPPER_COMPILER_EXTRA_LIBS@
+Cflags: -I${includedir}
```

Comment:
Might we also want the `${includedir}/mpp` here?  I wouldn't think so, but just making sure.

Reply:
Nope, that would be redundant, since the same headers are already installed in `${includedir}`.

Reply:
BTW, `pkg-config --cflags` is identical to `CFLAGS` and `pkg-config --libs` is identical to `LDFLAGS` generated by the `oshcc` wrapper.  So, if we ever find an issue here, both places need an update.